docs/src/3.3.Command.mdx

```diff
@@ -5,9 +5,9 @@ import CommandDemo from './jsx/Command.js';
 Command is the base class for all regl-based drawing commands. Subclasses should override getDrawProps and (and optionally getHitmapProps). Note: getHitmapProps is not implemented in this base class.
 Checkout [Writing A Custom Command Component](/guides/writing-a-custom-command-component) for more.
 
-| Name       | type    | Default | Description                                                                       |
-| ---------- | ------- | ------- | --------------------------------------------------------------------------------- |
-| layerIndex | ?number |         | defines the draw order of each command, e.g. -1 will put the shape in lower layer |
+| Name         | optional | type      | Default | Description                                                                       |
+| ------------ | -------- | --------- | ------- |
+| `layerIndex` | yes      | `?number` |         | defines the draw order of each command, e.g. -1 will put the shape in lower layer |
```

Comment:
Don't need the question mark any more if we explicitly note it's optional right?

Reply:
Our type definition wasn't quite strict before. Technically we allow `layIndex={null}` to be passed in since the flow definition is `layerIndex?: ?number`. But I think we should just have strict definition `layerIndex?: number` instead. Will update the Command component and docs both.